.pre-commit-config.yaml

```diff
@@ -31,3 +31,10 @@ repos:
   hooks: # check conformance to black formatting
    - id: black
      args: ['--diff', '-v'] # if run without arguments, will fail and will format the files
+
+- repo: https://github.com/blowekamp/demo_kwhooks
+  rev: v0.1.0
+  hooks:
+    - id: legacy-pre-commit
+    - id: legacy-prepare-commit-msg
+    - id: legacy-commit-msg
```

Comment:
I'd rather have the legacy/simple/bash hooks have a way to launch the python ones.  That way the weight of python is needed only when one wants to use it.

Reply:
yes the kw hooks are nice and light weight, the direct installation can still be done with the additional of the yaml configuration file. 

I started adding pre-commit with installation on top of the kw hooks. The pre-commit installation step renamed existing kw hooks in .git/hook with the addition of '-legacy" to their names and chained to execute them.

I suppose something like `python -mpre_commit  run --hook-stage pre-commit` could be used or added .hook-config.bash.